src/style.css

```
@@ -1,46 +1,112 @@
 body {
-  background-color:#f3f4f6;
+  background-color: #f3f4f6;
```

[Optional]

- [ ] This change will align items to the center of the page. The page will look better on mobile resolution.

```suggestion
  background-color: #f3f4f6;
  text-align: center;
```